server/spec/webhooks/github/delete_attributes_spec.rb

```diff
@@ -1,10 +1,7 @@
 require 'spec_helper'
 require 'helpers/github_payload_builder'
 
-include TxghServer
```

Comment:
Thanks for removing includes like these from all the spec files, what was I thinking? 😳